frontend/helper.js

```diff
@@ -0,0 +1,11 @@
+export const openBurgerMenuAndCloseMainHead = ({ event, setIsOpened, setIsOpenedMainHeader }) => {
```

Comment:
This method is only used in one component. Why did you put this in a separate file?

Reply:
We were taught not to interfere with js logic and jsx components